src/cli/Utils.cpp

```diff
@@ -135,6 +136,38 @@ namespace Utils
             compositeKey->addKey(fileKey);
         }
 
+#ifdef WITH_XC_YUBIKEY
+        if (!yubiKeySlot.isEmpty()) {
+            bool ok = false;
+            int slot = yubiKeySlot.toInt(&ok, 10);
+            if (!ok) {
+                err << QString("Invalid YubiKey slot %1").arg(yubiKeySlot) << endl;
+                return {};
+            }
+            if (slot != 1 && slot != 2) {
+                err << QString("Invalid YubiKey slot %1").arg(yubiKeySlot) << endl;
+                return {};
+            }
+
+            QString errorMessage;
+            bool blocking = YubiKey::instance()->isBlocking(slot, errorMessage);
+            if (!errorMessage.isEmpty()) {
+                err << errorMessage << endl;
+                return {};
+            }
+
+            auto key = QSharedPointer<YkChallengeResponseKeyCLI>(new YkChallengeResponseKeyCLI(
+                slot,
+                blocking,
+                QString("Please touch the button on your YubiKey to unlock " + databaseFilename),
+                QString("Challenge-response with YubiKey " + QString::number(slot) + " succeeded!!!"),
+                QString("Failed to unlock " + databaseFilename + " with YubiKey " + QString::number(slot)),
```

Comment:
This error should be handled by the database unlock process, not the yubikey key interface.